docs/services/domain-procedures/which-tlds-does-hypernode-offer.md

```diff
@@ -59,4 +61,4 @@ redirect_from:
 
 ## How to order
 
-Go to this page on how to order your domains in the Control Panel.
+Go to [this page](/services/domain-procedures/how-to-use-domains-in-the-control-panel.html#registering-a-new-domain) on how to order your domains in the Control Panel.
```

Comment:
```suggestion
Go to [this page](how-to-use-domains-in-the-control-panel.html#registering-a-new-domain) on how to order your domains in the Control Panel.
```

Reply:
Since it's in the same directory, you can skip the `/services/domain-procedures/` part.